spec/haproxy/templates/haproxy_config/backend_http_spec.rb

```diff
@@ -8,23 +8,23 @@
   end
 
   let(:properties) { {} }
-  let(:backend_http1) { haproxy_conf['backend http-routers-http1'] }
-  let(:backend_http2) { haproxy_conf['backend http-routers-http2'] }
+  let(:backend_http_one) { haproxy_conf['backend http-routers-http1'] }
```

Comment:
Was this mandated by a linter? Not all linter rules are useful. We may choose to instead ignore this rule instead of sweeping name changes.